src/parsers/instructions.ts

```diff
@@ -36,7 +36,11 @@ function handleTweetObject(obj: any, config: Config, promoted: boolean) {
 	for (const key of UserObjectPath) {
 		if (ptr.hasOwnProperty(key)) {
 			ptr = ptr[key];
-			if (ptr.__typename == 'Tweet' && ptr?.note_tweet?.is_expandable == true) {
+			if (
+				ptr.__typename == 'Tweet' &&
+				(ptr?.note_tweet?.is_expandable == true ||
+					typeof ptr?.edit_control?.edit_tweet_ids?.initial_tweet_id == 'string')
```

Comment:
hm, this is all it takes? Interesting...

Reply:
I believe so, if a tweet hasn't been edited that property doesn't exist. So I think `ptr?.edit_control?.edit_tweet_ids?.initial_tweet_id ` would evaluate to undefined.

There's also a list of tweet ids at `ptr.edit_control.edit_tweet_ids` that seems like it's always present, and will only have a length greater than 1 if the tweet was edited.

Reply:
ok, cool